lib/galaxy/tools/parameters/validation.py

```diff
@@ -559,6 +560,42 @@ def validate(self, value, trans=None):
             super().validate(isinstance(value, model.DatasetInstance) and not missing, value_to_show=missing)
 
 
+class MetadataEqualsValidator(Validator):
```

Comment:
I'm late, but maybe `MetadataEqualValidator` instead of `MetadataEqualsValidator` for consistency?

Reply:
Not to late https://github.com/galaxyproject/galaxy/pull/16489 :)

Thanks!